app/pages/project/classify.cjsx

```diff
@@ -142,7 +143,8 @@ module.exports = React.createClass
         sort: 'cellect' unless SKIP_CELLECT
 
       fetchSubjects = apiClient.type('subjects').get(subjectQuery).then (subjects) ->
-        upcomingSubjects.forWorkflow[workflow.id].push subjects...
+        nonLoadedSubjects = (newSubject for newSubject in subjects when newSubject isnt subjectToLoad)
```

Comment:
I may just be behind the coffeescript times, but aren't two objects never equal unless they're references to the same object, should this compare ids?